src/redux/reducers/userReducer.js

```diff
@@ -49,6 +53,20 @@ export default createReducer(intialState, {
   [logoutError]: (state, { payload }) => ({
     ...state,
     error:
-      payload?.full_messages?.[0] || 'An error occurred. Please try again.' 
+      payload?.full_messages?.[0] || 'An error occurred. Please try again.'
+  }),
+  [updatePasswordSuccess]: (state, { payload }) => ({ value: payload }),
+  [updatePasswordError]: (state, { payload }) => ({
+    ...state,
+    error:
+      (payload && payload.full_messages && payload.full_messages[0]) ||
+      'An error occured. Please try again.'
+  }),
+  [deleteAccountSuccess]: () => ({
+    intialState
+  }),
+  [deleteAccountError]: (state, { payload }) => ({
+    ...state,
+    error: payload?.full_messages?.[0] || 'An error occurred. Please try again.'
```

Comment:
I see that you are using this condition `payload?.full_messages?.[0] || 'An error occurred. Please try again.'` more than once, so maybe you can create a function for this with the payload as a param.

Reply:
Great idea! I'll add these changes at a new PR